CONTRIBUTING.md

```diff
@@ -1,5 +1,8 @@
 # Contributing
 
+This project adheres to the [Open Code of Conduct][code-of-conduct]. By participating, you are expected to uphold this code.
```

Comment:
Ah, I didn't realize that there was recommended text for applying the code. In that case, https://github.com/todogroup/opencodeofconduct/issues/16.